samples/04_gis_analysts_data_scientists/determining_site_suitability_for_oil_palm_plantation.ipynb

```diff
@@ -0,0 +1,639 @@
+{
```

Comment:
<p>The <code>input_raster</code>  parameter is not clear to me. How are you able to point to the same folder but copy different files each time? Is the <code>output_name</code>  used to determine what input file is picked to be published?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>